modules/examples/src/main/java/com/ibm/cloud/platform_services/case_management/v1/CaseManagementExamples.java

```diff
@@ -105,7 +105,7 @@ public static void main(String[] args) throws Exception {
       Response<Case> response = service.createCase(createCaseOptions).execute();
       Case xCase = response.getResult();
 
-      System.out.println(xCase);
+      System.out.printf("createCase() response: %n %s %n", xCase.toString());
```

Comment:
Same thing about the indentation that I've mentioned in the Go and Python repositorties:
```
uploadFile() response: 
 {
  "id": "xxx",
  "filename": "example.log",
  "size_in_bytes": 42,
  "created_at": "2021-03-23 17:04:18",
  "url": "xxx"
} 
```
```suggestion
      System.out.printf("createCase() response: %n%s%n", xCase.toString());
```

Reply:
I agree.  In your Go PR I mentioned the need to be consistent with the wording across all 4 SDKs.  So let's use "result" in the label, like this:
```
System.out.printf("createCase() result: %n%s%n", xCase.toString());
```